paper-checkbox.html

```diff
@@ -40,9 +40,12 @@
 `--paper-checkbox-checked-ink-color` | Selected/focus ripple color when the input is checked | `--primary-color`
 `--paper-checkbox-checkmark-color` | Checkmark color | `white`
 `--paper-checkbox-label-color` | Label color | `--primary-text-color`
+`--paper-checkbox-label-checked-color` | Label color when the input is checked | (none)
```

Comment:
I think for `none` you can just use `` (similar to what we do for mixins), or use the same default as `paper-checkbox-label-color `, which is probably more correct